flatdata-rs/lib/src/multiarrayview.rs

```diff
@@ -62,8 +61,9 @@ where
     ///
     /// Panics if index is greater than or equal to `MultiArrayView::len()`.
     pub fn at(&self, index: usize) -> MultiArrayViewItemIter<'a, Ts> {
-        let start = <Idx as IndexStruct>::index(self.index.at(index));
-        let end = <Idx as IndexStruct>::index(self.index.at(index + 1));
+        let start = <<Ts as VariadicStruct<'a>>::Index as IndexStruct>::index(self.index.at(index));
+        let end =
+            <<Ts as VariadicStruct<'a>>::Index as IndexStruct>::index(self.index.at(index + 1));
```

Comment:
Do you think it makes sense to store a part of this statement into a type alias? What clippy says actually?

Reply:
Doesn't seem possible with the current Rust version (since the alias would depend on some outer paramters)

Reply:
Thanks for trying!